make.go

```diff
@@ -236,12 +236,16 @@ func createGitRepository(debsrc, gopkg, orig string) (string, error) {
 		return dir, err
 	}
 
-	if err := runGitCommandIn(dir, "config", "user.name", getDebianName()); err != nil {
-		return dir, err
+	if debianName := getDebianName(); debianName == "TODO" {
```

Comment:
This patch fixes the problem for github.com/golang/example, but changing == to != does not, suggesting that this patch should not be submitted for now.